tests/testthat/test-tar_renv.R

```diff
@@ -1,28 +1,34 @@
 tar_test("tar_renv() works", {
   expect_false(file.exists("_packages.R"))
-  tar_option_set(packages = c("tibble", "qs"))
-  tar_renv(ask = FALSE)
+  tar_script({
+    tar_option_set(packages = c("tibble", "qs")); tar_pipeline()
+    })
+  tar_renv()
```

Comment:
Would you set `callr_function = NULL` for all but one call to `tar_renv()` in these tests? Otherwise, testing time adds up and we lose credit for code coverage.